index.html

```diff
@@ -7,5 +7,40 @@
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <div class="wrapper">
+      <header class="top">
+        <a class="active" href="#">Logo</a>
+        <nav class="flex-nav">
+          <a href="#" class="toggleNav">☰</a>
+        </nav>
+      </header>
+      <main>
+        <section class="backimg">
+          <div class="title">
+            <h1>I'm Ricardo glad to see you</h1>
+          </div>
+          <div class="para">
+            <p>
+              I’m a software developer! I can help you build a product, feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <div class="title2">
+            <p>LET'S CONNECT</p>
+          </div>
+        </section>
+
+        <div class="icons">
+          <img src="/img/twitter.svg" alt="Twitter logo" />
+          <img src="/img/github.svg" alt="github logo" />
+          <img src="/img/Linkedin.svg" alt="Linkedin logo" />
+          <img src="/img/look.svg" alt=" logo" />
+          <img src="/img/medium.svg" alt="medium logo" />
+        </div>
```

Comment:
- [ ] This section tag should cover all of these icons as well

Reply:
Now the section tag cover icons as well. well done!